services/node_create.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.consensus">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * See [NodeService.createNode()](#proto.NodeService)
+ */
+message NodeCreateTransactionBody {
+    /**
+     * node account id, mandatory field
+     */
+    AccountID account_id = 1;
+
+    /**
+     * description of the node, optional field
+     */
+    string description = 2;
+
+    /**
+    * inbound ip address or FQDN and port, mandatory field
+    */
+    ServiceEndpoint inbound_gossip_endpoint = 3;
+
+    /**
+     * outbound ip address or FQDN and port, mandatory field
+     */
+    ServiceEndpoint outbound_gossip_endpoint = 4;
+
+    /**
+    * # A node's service IP addresses and ports, optional field
+    */
+    repeated ServiceEndpoint service_endpoint = 5;
```

Comment:
Yes, this field is optional was confirmed in one of the meeting and #dynamic-address. repeated here because it could have a list of service_endpoints, not just one.